app/src/main/res/values/strings.xml

```diff
@@ -4,12 +4,16 @@
     <string name="title_recommend">Recommend</string>
     <string name="hint_text_search">Search music …</string>
     <string name="title_song_name">Name Song</string>
-    <string name="titile_new_release">New Realse</string>
-    <string name="titile_show_all">Show all</string>
-    <string name="titile_genres">Genres</string>
-    <string name="tittle_genres">Name of Genres</string>
-    <string name="tittle_descreption_genres">descreption</string>
-    <string name="tittle_play">PLAY</string>
+    <string name="title_new_release">New Realse</string>
+    <string name="title_show_all">Show all</string>
+    <string name="title_genres">Name of Genres</string>
+    <string name="title_description_genres">description</string>
+    <string name="title_play">PLAY</string>
+    <string name="string_add_mymusic">Add to my music</string>
+    <string name="string_add_playlist">Add to playlist</string>
+    <string name="string_play_next">Play next</string>
+    <string name="title_up_next">Up next</string>
+    <string name="title_genres_name">Genres</string>
     <string name="string_exam_curenttime">0:00</string>
```

Comment:
Không nên khai báo nhưng example này trong string.

Reply:
nên khai báo ntn a ?

Reply:
Đối với những text chỉ với mục đich là example để preview em có thể dùng tools:text="text sample"

Reply:
tools:text="toolsText" is used only for Android Studio layout preview, it not shows text when you run an app. It's perfect, when you don't want to set a text until you do some calculations, and only then set text programmatically by yourself in activity, also you want to see preview layout when you write code in Android Studio, how it will look like when you run an app.

android:text="androidText" is used to set text to a a layout element, textView or a button etc.